src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAO.scala

```diff
@@ -93,4 +97,22 @@ class ElasticSearchDAO(servers:Seq[Authority], indexName: String) extends Search
       case e: Exception => logger.warn(s"ES index '%s' could not be recreated and may be in an unstable state.".format(indexName))
     }
   }
+
+  override def findDocuments(criteria: LibrarySearchParams) : LibrarySearchResponse = {
+    val searchStr =
+      criteria.searchTerm match {
+        case None => ESQuery(Left(new ESMatchAll)).toJson.prettyPrint
+        case Some(searchTerm:String) => ESQuery(Right(ESWildcardObj(searchTerm))).toJson.prettyPrint
+      }
+    val searchReq = client.prepareSearch(indexName).setQuery(searchStr)
+    searchReq.setFrom(criteria.from.getOrElse(LibrarySearchConstants.defaultFrom))
+    searchReq.setSize(criteria.size.getOrElse(LibrarySearchConstants.defaultSize))
```

Comment:
nitpick: for syntax cleanliness, could this could be:
```
    val searchReq = client.prepareSearch(indexName).setQuery(searchStr)
        .setFrom(criteria.from.getOrElse(LibrarySearchConstants.defaultFrom))
        .setSize(criteria.size.getOrElse(LibrarySearchConstants.defaultSize))
```
this is pure style - so if you have preference in the other direction, Im ok.